Tribler/community/doubleentry/database.py

```diff
@@ -0,0 +1,266 @@
+""" This file contains everything related to persistence for DoubleEntry.
+"""
+import base64
+
+from os import path
+
+from Tribler.dispersy.database import Database
+
+
+def encode_db(db_object):
```

Comment:
This method is just plain lazy. If you're inserting binary into the database wrap it into a buffer object, if not don't.
You should be able to know the datatypes beforehand.